tensorflow/python/kernel_tests/data_structures/lookup_ops_test.py

```diff
@@ -3354,6 +3355,32 @@ def testMutableHashTable(self, is_anonymous):
     self.assertAllEqual([b"brain", b"salad", b"surgery"], sorted_keys)
     self.assertAllEqual([0, 1, 2], sorted_values)
 
+  @unittest.expectedFailure
+  @test_util.run_v2_only
+  def testImportedHashTable(self, is_anonymous):
+    # (TODO #24439) 
+    # Confirm unfixed https://github.com/tensorflow/tensorflow/issues/24439
```

Comment:
@mihaimaruseac It could be nice if we could standardize these two comments lines so that it will be easier to find `contribution welcome` tests to fix and the related ticket where I hope we could collected some hints for the contributors to prepare a PR and fix an `expectedFailure` test. 
Can you talk internally with your team?